src/store/metadata/stub/content/editions/additions-to-the-housing-stock_2022-2023.json

```diff
@@ -0,0 +1,163 @@
+  
+        ],
+        "spatial_coverage": "E92000001",
+        "temporal_resolution": [
+          "P1Y"
+        ],
+        "temporal_coverage": {
+          "start": "2022-01-01T00:00:00Z",
+          "end": "2023-12-31T00:00:00Z"
+        },
+      "versions_url": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock/editions/2022-2023/versions",
+      "versions": [
+        {
+          "@id": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock/editions/2022-2023/versions/1",
+          "issued": "2023-05-05T08:59:11Z"
+        }
+      ],
+      "next_release": "2024-04-19T09:30:00Z",
+      "table_schema": {
+        "columns": [
+          {
+              "name": "Period",
```

Comment:
name in all column mapping everywhere should be _snake_case_, like in `apprenticeships-achievements_2023-05.json`
"name": "period"